refextract/references/kbs.py

```diff
@@ -42,6 +43,23 @@
 from ..documents.text import re_group_captured_multiple_space
 
 
+@contextlib.contextmanager
+def file_resolving(fpath, reader=None, **kwargs):
+    if isinstance(fpath, six.string_types):
+        print('Load kb path %s' % fpath)
```

Comment:
I agree, although, in this PR @zzacharo was just refactoring. So moving to use the logging library would be the right thing to do, (either in a separate PR or at least a separate commit).

Note however that print is many more files...